src/modules/markets/container.js

```diff
@@ -33,6 +37,7 @@ const mapDispatchToProps = dispatch => ({
   clearMarketsFilteredSorted: () => dispatch(clearMarketsFilteredSorted()),
   toggleFavorite: marketId => dispatch(toggleFavorite(marketId)),
   loadMarketsInfo: marketIds => dispatch(loadMarketsInfo(marketIds)),
+  updateModal: modal => dispatch(updateModal(modal)),
 })
 
```

Comment:
move to src/modules/market/containers/market-properties.js if these properties are only needed at market-properties level. If not ping me, I'm missing something.